napari/components/_tests/test_multichannel.py

```diff
@@ -94,6 +94,7 @@
 ]
 
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
why this ignore?

Reply:
This is becaues test uses `np.random.random` to generate data that always returns floats